pandas/io/formats/latex.py

```diff
@@ -106,14 +110,9 @@ def pad_empties(x):
             raise AssertionError('column_format must be str or unicode, '
                                  'not {typ}'.format(typ=type(column_format)))
 
-        if not self.longtable:
-            buf.write('\\begin{{tabular}}{{{fmt}}}\n'
-                      .format(fmt=column_format))
-            buf.write('\\toprule\n')
-        else:
-            buf.write('\\begin{{longtable}}{{{fmt}}}\n'
-                      .format(fmt=column_format))
-            buf.write('\\toprule\n')
+        buf.write(self._build_latex_begin_env(column_format))
```

Comment:
i think this would be more clear to leave the if self.longtable and have a 
``self._format_begin_longttable`` and so on for each of end and table

Reply:
ok.  You are referring to the methodology of commit [911a491](https://github.com/pandas-dev/pandas/pull/25437/commits/911a49127adb2d5ec1b56dea97a448430b28eace) right?

Reply:
well, I suppose. This needs to be more modular and more clear to a casual reader. A giant if/then is not desired here. well named small function that do 1 think are good.